corrcal/corrcal.py

```diff
@@ -1,141 +1,100 @@
-import ctypes
 import time
-import site
-from glob import glob
 
 import numpy as np
 
+from .wrapper import *
 
-__all__ = ['sparse_2level', 'get_chisq', 'get_chisq_dense', 'get_gradient',
+__all__ = ['Sparse2Level', 'get_chisq', 'get_chisq_dense', 'get_gradient',
            'get_gradient_dense', 'chol', 'many_chol', 'tri_inv', 'many_tri_inv',
            'read_sparse', 'make_uv_grid', 'grid_data', 'mymatmul',
            'mult_vecs_by_blocks', 'make_uv_from_antpos']
 
-
+# Not really sure what this does.
```

Comment:
This allows you to use pyfof (a friends of friends implementation in python) to find redundant groups.
If pyfof is not installed the redundant grouping function "grid_data" cuts the uv-plane into a grid and selects baselines that fall into a grid cell as a redundant group.

See grid_data in wrapper.py

Reply:
Added these as a comment block in https://github.com/UKZN-Astronomy/corrcal/pull/21/commits/635a6eed3f47cf2b2155b6c109869b9f4f8ffc0a